src/lib/Controller/api/connector.c

```diff
@@ -168,3 +168,16 @@ int wb_connector_get_presence(WbDeviceTag tag) {
   robot_mutex_unlock_step();
   return result;
 }
+
+bool wb_connector_get_state(WbDeviceTag tag) {
+  bool result = false;
+  robot_mutex_lock_step();
+  Connector *con = connector_get_struct(tag);
+  if (con) {
+    result = con->is_locked;
+  } 
+    else
+    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
```

Comment:
```suggestion
  if (con)
    result = con->is_locked;
  else
    fprintf(stderr, "Error: %s(): invalid device tag.\n", __FUNCTION__);
```

Just for the reference, here is our coding style: https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style

Reply:
Thank you, changes have been applied